### PR TITLE
Add is_running method to Loop

### PR DESCRIPTION
**Pull request by Gobot1234**
### Summary
Adds a property to the `ext.tasks.Loop` class enabling someone to check if a task is running more easily.

### Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/ext/tasks/__init__.py [hunk 1]
```diff
@@ -276,6 +276,10 @@ def failed(self):
         """
         return self._has_failed
 
+    def is_running(self):
+        """:class:`bool`: Check if the task is currently running."""
+        return not self._task.done()
```
Comment (PikalaxALT) [inline discord/ext/tasks/__init__.py]: This needs to handle the case where `self._task is None`

Comment (Gobot1234) [reply discord/ext/tasks/__init__.py]: Will do

**Comment by tylergibbs2** [comment]
I don't see the necessity of this. As documented, the `get_task()` method returns `None` if there is no task running. If anything, your code can be simplified to `return bool(self._task)`.

I believe returning only `True` or `False` in this case is required, rather than including `None` in the available return types, as this is an `is_` method.

**Comment by Phxntxm** [comment]
You have misunderstood the docs... If the task has *not been ran* yet then yes, that will return `None`. However, if it has and the task has stopped, it will not return `None`... Instead it will return the task in its stopped/failed state

**Comment by Rapptz** [comment]
LGTM.

**Pull request closed without merging**